src/app.js

```diff
@@ -50,14 +51,17 @@ class App extends Component {
 
   render() {
     const {hash} = getLocationParts();
-    const {error, name, projectMap} = this.state;
+    const {description, error, name, projectMap} = this.state;
 
     return (
       <div className="app">
         <div className="error">{error}</div>
         {hash === 'display' ?
           <DataDisplay projectMap={projectMap} /> :
-            <DataEntry name={name} />}
+          <DataEntry // eslint-disable-line react/jsx-indent
```

Comment:
my linter was throwing an error there.  I wondered if that is a rule that should be turned off in my .eslintrc file.  I'm not sure but it wants to indent it on that ternary.